helm/cluster-vsphere/values.yaml

```diff
@@ -142,152 +70,61 @@ internal:
-    # eponymous property value in the template from which the virtual machine is
-    # cloned.
-    # <integer>
-    # diskGiB: 0
-
-    # FailureDomain is the failure domain unique identifier this Machine should
-    # be attached to, as defined in Cluster API. For this infrastructure
-    # provider, the name is equivalent to the name of the VSphereDeploymentZone.
-    # <string>
-    # failureDomain: ""
-
-    # Folder is the name or inventory path of the folder in which the virtual
-    # machine is created/located.
-    # <string>
-    # folder: ""
-
-    # HardwareVersion is the hardware version of the virtual machine. Defaults to
-    # the eponymous property value in the template from which the virtual machine
-    # is cloned. Check the compatibility with the ESXi version before setting the
-    # value.
-    # <string>
-    # hardwareVersion: ""
+kubectlImage:
+  registry: gsoci.azurecr.io
+  name: giantswarm/kubectl
+  tag: 1.23.5
 
-    # MemoryMiB is the size of a virtual machine's memory, in MiB. Defaults to
-    # the eponymous property value in the template from which the virtual machine
-    # is cloned.
-    # <integer>
-    # memoryMiB: 0
+metadata:
+  preventDeletion: false
 
-    # Network is the network configuration for this machine's VM.
-    # <Object> -required-
+nodeClasses:
+  default:
+    cloneMode: "linkedClone"
+    diskGiB: 50
+    numCPUs: 6
+    memoryMiB: 16896
+    resourcePool: "*/Resources"
+    template: "flatcar-stable-3602.2.1-kube-v1.25.16-gs"
     network:
-      # Devices is the list of network devices used by the virtual machine.
-      # TODO(akutz) Make sure at least one network matches the ClusterSpec.CloudProviderConfiguration.Network.Name
-      # can contain also: searchDomains, mtu, macAddr, gateway4, routes, nameservers, etc
-      # check kubectl explain VSphereMachineTemplate.spec.template.spec.network.devices for docs
-      # <[]Object> -required-
       devices:
-        # DHCP4 is a flag that indicates whether or not to use DHCP for IPv4 on this
-        # device. If true then IPAddrs should not contain any IPv4 addresses.
-      - dhcp4: true
-        # NetworkName is the name of the vSphere network to which the device will be connected.
-        # <string> -required-
-        networkName: 'grasshopper-capv'
-
-      # PreferredAPIServeCIDR is the preferred CIDR for the Kubernetes API server
-      # endpoint on this machine
-      # preferredAPIServerCidr: ""
-
-      # Routes is a list of optional, static routes applied to the virtual machine.
-      # <[]Object>
-      # routes: []
-      #   # Metric is the weight/priority of the route.
-      #   # <integer> -required-
-      # - metric: 50
-      #   # To is an IPv4 or IPv6 address.
-      #   # <string> -required-
-      #   to: ""
-      #   # Via is an IPv4 or IPv6 address.
-      #   # <string> -required-
-      #   via: ""
-
+      - networkName: 'grasshopper-capv'
```

Comment:
is it worth removing this default from the chart? this is very specific to our infra and wouldn't work for anyone else

Reply:
Good point

Reply:
https://github.com/giantswarm/cluster-vsphere/pull/177/commits/ef1f8da4e7eeb44eb93b84fa1b611aed05752613